bin/node-template/node/Cargo.toml

```diff
@@ -28,6 +28,8 @@ sc-service = { version = "0.10.0-dev", path = "../../../client/service" }
 sc-telemetry = { version = "4.0.0-dev", path = "../../../client/telemetry" }
 sc-transaction-pool = { version = "4.0.0-dev", path = "../../../client/transaction-pool" }
 sc-transaction-pool-api = { version = "4.0.0-dev", path = "../../../client/transaction-pool/api" }
+sc-offchain = { version = "4.0.0-dev", path = "../../../client/offchain" }
+sc-statement-store = { version = "4.0.0-dev", path = "../../../client/statement-store" }
```

Comment:
node-template not integrate statement-store, so this seems not necessary.

Reply:
Good find! Ty